src/pages/resolve/ResolvePortal.vue

```diff
@@ -0,0 +1,169 @@
+            <router-link to="/resolve/">
+              <q-tab name="home" icon="home" label="Home" />
+            </router-link>
+            <router-link to="/resolve/elections">
+              <q-tab name="mails" icon="mail" label="Elections" />
+            </router-link>
+            <q-tab name="movies" icon="movie" label="Movies" />
+          </q-tabs>
+        </div>
+      </div>
+    </div> -->
+    <router-view></router-view>
+  </div>
+</template>
+
+<script>
+import {
+  fetchArbConfig,
+  fetchArbitrators,
+  fetchElections,
+  fetchNominees,
+  fetchCaseFiles,
+  fetchOffers,
+} from "./util";
+
+export default {
+  data() {
+    return {
+      tab: "mails",
+      interval: null,
+    };
+  },
+  computed: {
+    getCurrentRouteName() {
+      if (this.$route.params && this.$route.params.id) {
+        return `${this.$route.name} ${this.$route.params.id}`;
+      }
+      return this.$route.name;
+    },
+  },
+  methods: {
+    async getArbConfig() {
+      try {
+        const config = await fetchArbConfig(this);
+        this.$store.commit("resolve/setArbConfig", config);
+      } catch (err) {
+        console.warn("fetchConfig error: ", err);
+      }
+    },
+    async getArbitrators() {
+      try {
+        const arbitrators = await fetchArbitrators(this);
+        this.$store.commit("resolve/setArbitrators", arbitrators);
+      } catch (err) {
+        console.warn("fetchArbitrators error: ", err);
+      }
+    },
+    async getElections() {
+      try {
+        const elections = await fetchElections(this);
+        this.$store.commit("resolve/setElections", elections);
+      } catch (err) {
+        console.warn("getElections error: ", err);
+      }
+    },
+    async getNominees() {
+      try {
+        const nominees = await fetchNominees(this);
+        this.$store.commit("resolve/setNominees", nominees);
+      } catch (err) {
+        console.warn("getNominees error: ", err);
+      }
+    },
+    async getCaseFiles() {
+      try {
+        const caseFiles = await fetchCaseFiles(this);
+        this.$store.commit("resolve/setCaseFiles", caseFiles);
+      } catch (err) {
+        console.warn("getCaseFiles error: ", err);
+      }
+    },
+    async getOffers() {
+      try {
+        const offers = await fetchOffers(this);
+        this.$store.commit("resolve/setOffers", offers);
+      } catch (err) {
+        console.warn("getOffers error: ", err);
+      }
+    },
+    fetchAllData() {
+      this.getArbConfig();
+      this.getArbitrators();
+      this.getElections();
+      this.getNominees();
+      this.getCaseFiles();
+      this.getOffers();
+    },
+  },
+  mounted() {
+    this.fetchAllData();
+    this.interval = setInterval(this.fetchAllData, 10000);
+  },
+  unmounted() {
+    clearInterval(this.interval);
+  },
+};
+</script>
+
+<style lang="scss">
+@import url("../../css/app.sass");
+
+.q-tooltip {
+  font-size: 1rem;
+  line-height: 1rem;
+}
+</style>
+
+<style scoped lang="scss">
```

Comment:
style sections should be global always. Verify your styles don't break anything else when they get free.

```suggestion
<style lang="scss">
```